tests/test_git_parser.py

```diff
@@ -66,13 +66,23 @@ def test_calculate_loc_changes(repository_paths: dict[str, pathlib.Path]) -> Non
     """
     Test the calculate_loc_changes function.
     """
-    for _, repo_path in repository_paths.items():
-        source_commit, target_commit = get_most_recent_commits(repo_path)
-
-        loc_changes = calculate_loc_changes(repo_path, source_commit, target_commit)
+    file_sets = {
+        "high_entropy": ["high_entropy2.md", "high_entropy.md"],
+        "low_entropy": ["low_entropy.md"],
+    }
 
-        # Ensure the dictionary is not empty
-        assert loc_changes
+    results = {}
 
-        # Ensure the total lines changed is greater than zero
-        assert sum(loc_changes.values()) > 0
+    for label, repo_path in repository_paths.items():
+        source_commit, target_commit = get_most_recent_commits(repo_path)
+        loc_changes = calculate_loc_changes(
+            repo_path, source_commit, target_commit, file_sets[label]
+        )
+        results[label] = loc_changes
+
```

Comment:
Consider adding another check to make sure that the file sets are different from one another.

Reply:
Referenced in test dataset issue